etl_framework/MySqlDatabase.py

```diff
@@ -1 +1,2 @@
-raise Exception("Class has moved. Import SqlDatabaseInterface from etl_framework.datastore_interfaces instead!!")
+raise Exception("Class has moved. Import `SqlDatabaseInterface` from \
+    `etl_framework.datastore_interfaces.sql_database_interface instead`!")
```

Comment:
@michaelliu2 Wait, we want to import `MySqlDatabase` and not `SqlDatabaseInterface`?